components/scream/src/share/grid/remap/abstract_remapper.hpp

```diff
@@ -178,9 +218,10 @@ class AbstractRemapper
   int           m_num_fields;
   int           m_num_registered_fields;
 
-  // Whether fields have been provided. Recall that the user may register
-  // fields passing only an identifier, and actually bind fields only
-  // at a later moment.
+  // This vector stores a boolean whose elements are the indices of fields that
```

Comment:
"This vector stores a boolean whose elements..." seems confusing. Maybe "A vector of booleans, whose elements ..."? I don't know, it sounds like 'whose' refer to the boolean(s) rather than the vector.

Reply:
Good catch. I obviously did some mid-sentence editing that I didn't clean up. :-)